next.config.js

```diff
@@ -62,4 +62,11 @@ const plugins = [
 module.exports = withPlugins(plugins, {
     reactStrictMode: true,
     output: 'export',
+    async redirects() {
```

Comment:
Does `redirects` method work after `next export`?

Reply:
I've removed this part in favour of getStaticProps https://github.com/gravity-ui/landing/pull/45/commits/cc9e761e8c7266543d3fe72de983968a1742572a